app/assets/stylesheets/modules/_footer.scss

```diff
@@ -4,4 +4,7 @@ footer {
   padding-top: 20px;
   padding-bottom: 10px;
   background-color: #41679a;
-}
\ No newline at end of file
+  a, a:hover {
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.